lib/nudge_web/controllers/site_controller.ex

```diff
@@ -36,11 +36,15 @@ defmodule NudgeWeb.SiteController do
 
     case Accounts.update_site(site, %{active: active}) do
       {:ok, _site} ->
-        redirect(conn, to: Routes.site_path(conn, :index))
-
-      {:error, %Ecto.Changeset{} = changeset} ->
+        if site.active == true do
```

Comment:
Shouldn't this be opposite, the toggle would change the `site` after update
Otherwise you would need to get the updated site from the `{:ok, updated_site}`